project-fortis-interfaces/src/components/Insights/Maps/HeatMap.js

```diff
@@ -77,8 +78,12 @@ export default class HeatMap extends React.Component {
   componentWillReceiveProps(nextProps) {
     const { placeid, defaultZoom } = this.state;
     const { targetBbox } = this.props;
-
-    if (hasChanged(this.props, nextProps) && nextProps.selectedplace.placeid && placeid !== nextProps.selectedplace.placeid) {
+    const { dashboardIsLoadedFromShareLink } = nextProps;
+    if (hasChanged(this.props, nextProps) && dashboardIsLoadedFromShareLink && !this.state.sharedLinkMapRepositions) {
+      this.refs.map.leafletElement.fitBounds([this.getMapBounds(nextProps.bbox)][0]);
```

Comment:
Isn't `[this.getMapBounds(nextProps.bbox)][0]` the same as `this.getMapBounds(nextProps.bbox)`?